dropwizard-configuration/src/main/java/io/dropwizard/configuration/YamlConfigurationFactory.java

```diff
@@ -105,9 +105,9 @@ public T build(ConfigurationSourceProvider provider, String path) throws IOExcep
     @Override
     public T build() throws IOException, ConfigurationException {
         try {
-            final JsonNode node = mapper.valueToTree(klass.getConstructor().newInstance());
+            final JsonNode node = mapper.valueToTree(klass.newInstance());
```

Comment:
I was hoping this would be squashed when it's merged in using the GitHub squash button. I can update the description later today.